File: Exo3.1.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[])
{
    if (argc != 3) 
    {
        printf("Usage : %s nombre1 nombre2\n", argv[0]);
        return (1);
    }

    int a = atoi(argv[1]);  // Convertir le premier argument en entier
    int b = atoi(argv[2]);  // Convertir le deuxième argument en entier

    int somme = a + b;

    printf("La somme de %d et %d est %d\n", a, b, somme);

    return (0);
}

File: Exo3.1.py
import sys

if len(sys.argv) != 3:
    print(f"Usage: {sys.argv[0]} nombre1 nombre2")
    sys.exit(1)

a = int(sys.argv[1])
b = int(sys.argv[2])

somme = a + b

print(f"La somme de {a} et {b} est {somme}")